README.md

```diff
@@ -42,15 +42,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/29td)
+- Twitter: [@twitterhandle](https://twitter.com/NonhlanhlaMnde6)
```

Comment:
I recommend you change `Author1` to your full name, it'll make your repo more professional. I'd also recommend you add your LinkedIn profile if you have one, that's the first thing recruiters and potential employers will look for when trying to contact you.